readme.txt

```diff
@@ -90,6 +90,35 @@ Source: https://stocksnap.io/photo/4B83RD7BV9
 
 == Changelog ==
 
+= 2.7.0 - 2023-03-21 =
+* Tweak: Update `Requires at least 5.9`
+* Tweak: Update `Tested up to 6.2`
+* Tweak: Remove backwards compatibility support for `wp_body_open()`
+* Tweak: Match `search.php` markup to `archive.php` markup
+* Tweak: Check if posts have featured images set
+* Tweak: Remove unnecessary `role` attributes from HTML landmark elements
+* Tweak: Escape translation strings for secure HTML output
+* Tweak: Use i18n function to make the "Menu" string translatable
+* Tweak: Minify SVG assets
+* Tweak: Make header nav-menu keyboard accessible
+* Tweak: Add `role="button"` to the nav-menu toggle for better accessibility
+* Tweak: Toggle mobile nav-menu with `Enter` & `Space` keyboard keys
+* Tweak: Add `hello_elementor_enable_skip_link` filter to enable/disable the skip link
+* Tweak: Add `hello_elementor_skip_link_url` filter to change skip link URL
+* Tweak: Use theme CSS not Elementor plugins CSS
+* Tweak: Match CSS to the new Elementor Editor Facelift
+* Tweak: Added support for the new Elementor version
+* Tweak: Match autoprefixer browsers list settings to Elementor plugin
```

Comment:
Remove this line (not relevant)

Reply:
Replaced with:
* Tweak: Update autoprefixer to exclude dead browsers